### PR TITLE
Updates to ladder command

### PR DESCRIPTION
**Pull request by prayansh**
## minor fixes and QOL updates
* New messages
* Stop typing by default
* Ladder command does not break with >10 users

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -29,11 +29,10 @@ bot.on('message', function (message) {
     var discordName = message.member.displayName;
     var discordID = message.member.id.toString();
 
-    if (message.content.substring(0, 1) === '!') {
+    if (message.content.substring(0, 1) === '*') {
```
Comment (padster) [inline bot.js]: was this temporary, or changing to * to avoid collisions with other bots?

File: bot.js [hunk 2]
```diff
@@ -68,6 +67,9 @@ bot.on('message', function (message) {
                 break;
             }
             // Just add any case commands if you want to..
+            default: {
+                message.channel.stopTyping(true);
```
Comment (padster) [inline bot.js:72]: lol, nice, just to be sure.
probably safe to remove shutup and stfu too

File: command/ladder.js [hunk 3]
```diff
@@ -29,16 +29,25 @@ function run(discordName, discordID, message, args, onComplete) {
...
         // Next step, query for all their ranks in one go:
-        logger.info("Getting all stats for " + batchPayload.length + " users...");
-        rlClient.getStatsBatch(batchPayload).then(
-            function (userRatings) {
+        logger.info("Getting all stats for " + payload.length + " users...");
+        var statsPromises = [];
+        for (var batch of batchPayload) {
+            statsPromises.push(rlClient.getStatsBatch(batch));
+        }
+        Promise.all(statsPromises).then(
```
Comment (padster) [inline command/ladder.js:48]: sweet! I hadn't realized promises had an all(), this is a nice way to do it.

**Review by padster — COMMENTED** [review]
Calculated.

Looks good, this is an approval but I'm marking as 'comment' for now so it doesn't auto-merge, just in case the ! -> * wasn't intentional

**Review by padster — APPROVED** [review]

File: bot.js (hunk 2 above)
Comment (prayansh) [reply bot.js:72]: im just gonna keep them there as a fallback.

File: command/ladder.js (hunk 3 above)
Comment (prayansh) [reply command/ladder.js:48]: Yea! it's pretty neat!

**Pull request merged**